Misc/NEWS.d/next/Build/2019-06-18-08-58-30.bpo-35360.-CWbfy.rst

```diff
@@ -0,0 +1 @@
+Update OS X installer to use SQLite 3.28.0.
```

Comment:
Apple now uses "macOS" instead of "OS X" and we should, too.  Also, please put the NEWS entry in the "macOS" section rather than the "Build" section.